code/modules/projectiles/guns/energy/laser/cog.dm

```diff
@@ -43,7 +43,7 @@
 	allow_greyson_mods = FALSE
 
 /obj/item/gun/energy/cog/sprocket
-	name = "Soteria \"Sprocket\" lasgun"
+	name = "Soteria \"Sprocket\" laser carbine"
```

Comment:
As the person that coded the gun, since it's the only one who's got a stock modification, I'd say it's the only one that qualifies as a "carbine", otherwise the Cog is a laser **rifle** for all intents and purposes.